packages/webcomponents/src/ui-components/index.ts

```diff
@@ -1,10 +1,9 @@
 export * from './button';
 export * from './button-spinner';
 export * from './styled-host/styled-host';
-export * from './table/exported-parts';
-export * from './table-empty-state';
-export * from './table-error-state';
-export * from './table-loading-state';
+export * from './table/table-empty-state';
+export * from './table/table-error-state';
+export * from './table/table-loading-state';
 export * from './skeleton';
```

Comment:
Can we also export everything from the new `./table` file you created in this index file?